ReseauEntourage/entourage-android
# Add pre-onboarding and improve login UI tests

[Ticket](https://entourage-asso.atlassian.net/browse/EN-3661)

`LoginTest`:
- Update deprecated `ActivityTestRule` import to `ActivityScenarioRule`
- Ensure that MainActivity is launched when login is successful

`PreOnboardingTest`:
- Test full pre-onboarding journey (all pages seen) with next button or image swipe
- Test pre-onboarding skipped to login page
